src/main/kotlin/net/atos/zac/zaken/ZakenService.kt

```diff
@@ -12,48 +14,78 @@ import net.atos.client.zgw.zrc.model.RolOrganisatorischeEenheid
 import net.atos.client.zgw.zrc.model.Zaak
 import net.atos.client.zgw.ztc.ZTCClientService
 import net.atos.client.zgw.ztc.model.generated.RolType
+import net.atos.zac.event.EventingService
 import net.atos.zac.identity.model.Group
 import net.atos.zac.identity.model.User
+import net.atos.zac.websocket.event.ScreenEventType
 import net.atos.zac.zoeken.IndexeerService
 import net.atos.zac.zoeken.model.index.ZoekObjectType
-import java.util.UUID
+import java.util.*
```

Comment:
Start import !

Reply:
Yes I know. Our Detekt config still does not care about this and at least my current IntelliJ settings adds wildcards imports for some reason.

Reply:
Aha. For some reason this is a default in Detekt: https://ncorti.com/detekt/docs/rules/style#wildcardimport

`excludeImports (default: ['java.util.*'])`

No idea why.. I'll make a separate PR to fix this. After that we should also all update our IntelliJ settings. Not sure if we can automate this like we do for the Java Eclipse rules.